docs/_articles/en/index.md

```diff
@@ -3,41 +3,44 @@ title: Salesforce Code Analyzer Plug-In
 permalink: /
 lang: en
 ---
+
```

Comment:
I deleted `/v3.x/index.md` and copied its contents into this file, thereby making 3.x the default landing page.